freetextresponse/freetextresponse.py

```diff
@@ -242,6 +242,16 @@ def student_view(self, context=None):
         )
         return fragment
 
+    def max_score(self):
+        """
+        Returns the configured number of possible points for this component.
+        Arguments:
+            None
+        Returns:
+            float: The number of possible points for this component
+        """
+        return self.weight if self.has_score else None
```

Comment:
Based on other CAPA xblocks and their progress page, this should return the instructor weight.  Credit.full.value is always one.  If the weight is 3 for an xblock then the progress page available points goes up by 3 for that subsection.  

Per discussion, I'll update so max_score returns self.weight with no checks.